### ISSUE TITLE
Info shows null on all wifi properties

### ISSUE THREAD

**Issue by Flip76** [vangorra/python_gogogate2_api#9]
Another small issue I found:

"wifi" just shows null for each property:
  
"wifi": {
    "SSID": null,
    "linkquality": null,
    "signal": null
  }

**Comment by vangorra** [comment]
I'm curious if you're running an older or newer firmware version. My firmware version is 2.60 and api version is 1.5. My wifi info looks like this:
```
  "wifi": {
    "SSID": null,
    "linkquality": "79%",
    "signal": "-55 dBm"
  },
```

**Comment by Flip76** [comment]
Thanks for looking into it: My versions:

```
  "model": "ismartgateLITE"
  "apiversion": "3"
  "firmwareversion": "163"
  "newfirmware": false
```

1.63 is indeed the latest firmwareversion available for the ismartgate LITE devices...

**Comment by vangorra** [comment]
Interesting. It appears ismartgate does not provide wifi information but gogogate2 does. The only other method for retrieving this data is to screen scrape the HTML which is unreliable and messy.

Output from my ismartgate device.
```
...
                                <model>ismartgateLITE</model>
                                <apiversion>3</apiversion>
                                <ismartgatename>home</ismartgatename>
                                <remoteaccessenabled>yes</remoteaccessenabled>
                                <remoteaccess>UNIQUE_KEY.isgaccess.com</remoteaccess>
                                <firmwareversion>163</firmwareversion>
                                <newfirmware>no</newfirmware> 
...
                                <wifi>
                                        <SSID></SSID>
                                        <linkquality></linkquality>
                                        <signal></signal>
                                </wifi>
```

**Comment by Flip76** [comment]
I will contact the guys from Gogogate as recently I was in touch with one of their developers trying to fix a wifi issue (which turned out to be a faulty hardware). Maybe he can do a fix for the next firmware to display the wifi-information correctly...

**Comment by Flip76** [comment]
Oh... I just saw:

![image](https://user-images.githubusercontent.com/40655333/93024667-46fe8000-f5f8-11ea-8abf-79b406c4333a.png)

For some reason the ismartgate web-gui shows "connected via LAN" (but it's definitely on WiFi!). This might cause the wifi-data to be empty (even if in the settings I can see the wifi strength). Seems like the root cause is that I configured a static IP on the device and since it shows the wrong connection method.

But actually this is another good point to go back to the support guy :-)
